core/include/traccc/utils/relations.hpp

```diff
@@ -14,7 +14,7 @@
 
 namespace traccc {
 struct [[maybe_unused]] channel0_major_cell_order_relation{
-    template <typename T1, typename T2> TRACCC_HOST_DEVICE [[maybe_unused]] bool
+    template <typename T1, typename T2>[[maybe_unused]] TRACCC_HOST_DEVICE bool
```

Comment:
I'm not in love with the lack of a space in front of `[[maybe_unused]]`. :thinking: